### PR TITLE
Feature/cost center ud

### PR DESCRIPTION
**Pull request by jongamble**
## Why?
Need to be able to update and delete cost centers, as well as remove facilities associated with a cost center. Also, need to be able to get all cost centers for a certain organization.

## What changed?
Added Update and Delete cost center methods
Added remove facilities method
Added `getAllByOrganizationId` method

### CODE REVIEW

File: support/fixtures/factories/costCenter.js [hunk 1]
```diff
@@ -15,25 +14,6 @@ factory
     organizationId: () => factory.build('organization').id,
     updatedAt: () => faker.date.recent().toISOString()
   })
-  .attr('facilities', ['id', 'fromServer'], (id, fromServer) => {
```
Comment (jongamble) [inline support/fixtures/factories/costCenter.js]: keeping this consistent with facility_groupings

File: docs/CostCenters.md [hunk 2]
```diff
@@ -80,6 +84,29 @@ contxtSdk.facilities.costCenters
   .catch((err) => console.log(err));
 ```
 
+<a name="CostCenters+delete"></a>
+
+### costCenters.delete(costCenterId) ⇒ <code>Promise</code>
```
Comment (leeallen337) [inline docs/CostCenters.md]: Just curious but how do you do the arrow here? It's cool

File: docs/CostCenters.md [hunk 3]
```diff
@@ -80,6 +84,29 @@ contxtSdk.facilities.costCenters
...
+API Endpoint: '/costcenters/:costCenterId'
+Method: DELETE
+
+**Kind**: instance method of [<code>CostCenters</code>](#CostCenters)  
+**Fulfill**: <code>undefined</code>  
+**Reject**: <code>Error</code>
+
+| Param        | Type                | Description                                     |
+| ------------ | ------------------- | ----------------------------------------------- |
+| costCenterId | <code>string</code> | The id of the cost center (formatted as a UUID) |
```
Comment (leeallen337) [inline docs/CostCenters.md]: I've seen id capitalized and non-capitalized. Should we capitalize it here?

File: docs/CostCenters.md [hunk 4]
```diff
@@ -100,3 +127,81 @@ contxtSdk.facilities.costCenters
   .then((costCenters) => console.log(costCenters))
   .catch((err) => console.log(err));
 ```
+
+<a name="CostCenters+getAllByOrganizationId"></a>
+
+### costCenters.getAllByOrganizationId() ⇒ <code>Promise</code>
```
Comment (leeallen337) [inline docs/CostCenters.md]: Do we need to add an organization id to the argument here?

File: docs/CostCenters.md [hunk 5]
```diff
@@ -100,3 +127,81 @@ contxtSdk.facilities.costCenters
...
+API Endpoint: '/costcenters/:costCenterId/facility/:facilityId'
+Method: DELETE
+
+**Kind**: instance method of [<code>CostCenters</code>](#CostCenters)  
+**Reject**: <code>Error</code>
+
+| Param        | Type                | Description                           |
+| ------------ | ------------------- | ------------------------------------- |
+| costCenterId | <code>string</code> | UUID corresponding with a cost center |
+| facilityId   | <code>number</code> |                                       |
```
Comment (leeallen337) [inline docs/CostCenters.md]: Does this description need something similar to the add one?

File: docs/CostCenters.md [hunk 6]
```diff
@@ -100,3 +127,81 @@ contxtSdk.facilities.costCenters
...
+
+### costCenters.removeFacility(costCenterId, facilityId) ⇒ <code>Promise</code>
+
+Removes a facility from a cost center
+
+API Endpoint: '/costcenters/:costCenterId/facility/:facilityId'
+Method: DELETE
+
+**Kind**: instance method of [<code>CostCenters</code>](#CostCenters)  
+**Reject**: <code>Error</code>
```
Comment (leeallen337) [inline docs/CostCenters.md]: Do we need a fulfills here?

File: src/utils/facilities/formatCostCenterFromServer.spec.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import omit from 'lodash.omit';
 import formatCostCenterFromServer from './formatCostCenterFromServer';
 
-describe('utils/facilities/formatCostCenterFromServer', function() {
+describe.only('utils/facilities/formatCostCenterFromServer', function() {
```
Comment (leeallen337) [inline src/utils/facilities/formatCostCenterFromServer.spec.js]: Probably need to delete this `.only`

File: docs/CostCenters.md (hunk 2 above)
Comment (jongamble) [reply docs/CostCenters.md]: All of the `.md` files are created with rollup. Do an `npm run build` to generate the docs.

File: docs/CostCenters.md (hunk 3 above)
Comment (jongamble) [reply docs/CostCenters.md]: Yes we should! Good catch.

**Pull request merged**